pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/domain.go

```diff
@@ -55,10 +55,11 @@ func GenerateDomainName(name string, obj metav1.ObjectMeta, ingressConfig *v1bet
 		return "", fmt.Errorf("error rendering the domain template: %w", err)
 	}
 
-	urlErrs := validation.IsFullyQualifiedDomainName(field.NewPath("url"), buf.String())
-	if urlErrs != nil {
-		return "", fmt.Errorf("invalid domain name %q: %w", buf.String(), urlErrs.ToAggregate())
+	if !ingressConfig.DisableIngressCreation {
+		urlErrs := validation.IsFullyQualifiedDomainName(field.NewPath("url"), buf.String())
+		if urlErrs != nil {
+			return "", fmt.Errorf("invalid domain name %q: %w", buf.String(), urlErrs.ToAggregate())
```

Comment:
isn't better to return nil instead ""?

Reply:
The return type is string. This value is also what's used in `url.Host` which also requires a string.